GetMessage/__tests__/handler.test.ts

```diff
@@ -117,8 +117,13 @@ const aNewMessageWithoutContent: NewMessageWithoutContent = {
 
 const aRetrievedMessageWithoutContent: RetrievedMessageWithoutContent = {
   ...aNewMessageWithoutContent,
+  /* 
+  FIXME: these fields are shipped with RetrievedNotificationStatus objects
+  However, RetrievedNotificationStatus type doesn't include such fields in its definition
+  hence, the compiler consider such fields an error
   _self: "xyz",
-  _ts: 1,
+  _ts: 1, 
+  */
```

Comment:
@cloudify here is a dilemma: 
attributes such as `_self` are present in retrieved objects, but we actually never use them. The error above is due to this configuration: https://github.com/pagopa/io-functions-commons/blob/master/src/utils/cosmosdb_model.ts#L124

Should we refactor `CosmosdbModel` to extend `ResourceT`? I think such metadata is useless at user level, but still we have an incorrect structural mapping